processing/src/main/java/org/apache/druid/query/topn/PooledTopNAlgorithm.java

```diff
@@ -68,38 +68,38 @@
   @VisibleForTesting
   static void setSpecializeGeneric1AggPooledTopN(boolean value)
   {
-    PooledTopNAlgorithm.specializeGeneric1AggPooledTopN = value;
+    PooledTopNAlgorithm.SPECIALIZE_GENERIC_ONE_AGG_POOLED_TOPN = value;
     computeSpecializedScanAndAggregateImplementations();
   }
 
   @VisibleForTesting
   static void setSpecializeGeneric2AggPooledTopN(boolean value)
   {
-    PooledTopNAlgorithm.specializeGeneric2AggPooledTopN = value;
+    PooledTopNAlgorithm.SPECIALIZE_GENERIC_TWO_AGG_POOLED_TOPN = value;
     computeSpecializedScanAndAggregateImplementations();
   }
 
   @VisibleForTesting
   static void setSpecializeHistorical1SimpleDoubleAggPooledTopN(boolean value)
   {
-    PooledTopNAlgorithm.specializeHistorical1SimpleDoubleAggPooledTopN = value;
+    PooledTopNAlgorithm.SPECIALIZE_HISTORICAL_ONE_SIMPLE_DOUBLE_AGG_POOLED_TOPN = value;
     computeSpecializedScanAndAggregateImplementations();
   }
 
   @VisibleForTesting
   static void setSpecializeHistoricalSingleValueDimSelector1SimpleDoubleAggPooledTopN(boolean value)
   {
-    PooledTopNAlgorithm.specializeHistoricalSingleValueDimSelector1SimpleDoubleAggPooledTopN = value;
+    PooledTopNAlgorithm.SPECIALIZE_HISTORICAL_SINGLE_VALUE_DIM_SELECTOR_ONE_SIMPLE_DOUBLE_AGG_POOLED_TOPN = value;
     computeSpecializedScanAndAggregateImplementations();
   }
 
-  private static final Generic1AggPooledTopNScanner defaultGeneric1AggScanner =
+  private static final Generic1AggPooledTopNScanner DEFAULT_GENERIC1_AGG_SCANNER =
```

Comment:
This is inconsistent with other renames in this class which translate 1 to `_ONE_`. Also in 3 renames below.